tf-psa-crypto/tests/CMakeLists.txt

```diff
@@ -0,0 +1,362 @@
+set(libs
```

Comment:
This file is duplicating {MBEDTLS-ROOT}/tests/CMakeLists.txt.  It is effectively a large ammount of duplicated code. I am assuming those files are expected to diverge later on, so could we possibly source one from the other when they need to be identical?

Reply:
Eventually there will be one in mbedtls and one in TF-PSA-Crypto with quite similar contents and we do not plan to share anything there. Otherwise as of this PR, the two files diverge in the two following commits. I've done it that way to highlight in the commit history that the new tf-psa-crypto/tests/CMakeLists.txt file is just a derivation of the one we have in Mbed TLS and you can see the derivation in the commit "Add tf-psa-crypto test suites build".